tasks/e2e-simple.sh

```diff
@@ -76,51 +76,68 @@ if [ "$EXPECTED" != "$ACTUAL" ]; then
   exit 1
 fi
 
+if hash npm 2>/dev/null
```

Comment:
Revert all changes in this file, and keep it consistent with CRA5,
Tao Fang is working on how to build and test CRA5